### PR TITLE
Refactor capture resource and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by Diptipowervs** [comment]
--- PASS: TestAccIBMPICaptureBasic (67.31s)
PASS

File: ibm/service/power/ibm_pi_constants.go [hunk 1]
```diff
@@ -4,31 +4,38 @@ import "time"
...
+	Arg_CloudInstanceID                      = "pi_cloud_instance_id"
+	Arg_ImageName                            = "pi_image_name"
+	Arg_InstanceCaptureCloudStorageAccessKey = "pi_capture_cloud_storage_access_key"
+	Arg_InstanceCaptureCloudStorageImagePath = "pi_capture_storage_image_path"
+	Arg_InstanceCaptureCloudStorageSecretKey = "pi_capture_cloud_storage_secret_key"
+	Arg_InstanceCaptureCloudStorageRegion    = "pi_capture_cloud_storage_region"
+	Arg_InstanceCaptureDestination           = "pi_capture_destination"
+	Arg_InstanceCaptureName                  = "pi_capture_name"
+	Arg_InstanceCaptureVolumeIds             = "pi_capture_volume_ids"
+	Arg_InstanceName                         = "pi_instance_name"
```
Comment (michaelkad) [inline ibm/service/power/ibm_pi_constants.go:17]: All those should be name just like `Arg_InstanceName`
Prefix+Camel case
this: `Arg_InstanceCaptureCloudStorageAccessKey = "pi_capture_cloud_storage_access_key"` should be Arg_CaptureCloudStorageAccessKey = "pi_capture_cloud_storage_access_key"

File: ibm/service/power/ibm_pi_constants.go [hunk 2]
```diff
@@ -4,31 +4,38 @@ import "time"
...
+	Arg_SPPPlacementGroupPolicy              = "pi_spp_placement_group_policy"
+	Arg_SharedProcessorPoolHostGroup         = "pi_shared_processor_pool_host_group"
+	Arg_SharedProcessorPoolID                = "pi_shared_processor_pool_id"
+	Arg_SharedProcessorPoolName              = "pi_shared_processor_pool_name"
+	Arg_SharedProcessorPoolPlacementGroupID  = "pi_shared_processor_pool_placement_group_id"
+	Arg_SharedProcessorPoolReservedCores     = "pi_shared_processor_pool_reserved_cores"
+	Arg_SnapshotID                           = "pi_snapshot_id"
+	Arg_StoragePool                          = "pi_storage_pool"
+	Arg_StorageType                          = "pi_storage_type"
+	Arg_VTL                                  = "vtl"
```
Comment (michaelkad) [inline ibm/service/power/ibm_pi_constants.go:35]: This should be at the bottom

**Review by michaelkad — CHANGES_REQUESTED** [review]

**Comment by michaelkad** [comment]
Opened #128

**Pull request closed without merging**